<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>567</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_play">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/pause.png</normaloff>:/icons/pause.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_song">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>741</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>[歌曲名]</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_position">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_position">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>[播放时间]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_previous">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/previous.png</normaloff>:/icons/previous.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>320</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_playbackMode">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/loop.png</normaloff>:/icons/loop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_playlist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>331</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_lyrics">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>331</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
